src/container/RelayContainer.js

```diff
@@ -771,6 +771,7 @@ function createContainerComponent(
 
       const fragmentPointers = this._fragmentPointers;
       return (
+        !!nextState.relayProp.pendingVariables ||
```

Comment:
This could cause unnecessary re-renders if props are received while a fetch is pending. Let's make this a comparison between the pending variables on the previous and next state.